### ISSUE TITLE
Don't overfitting model

### ISSUE THREAD

**Issue by progred** [open-mmlab/mmdetection#2355]
Good day!

I tried to overfit model `faster_rcnn_r50_fpn_1x` on my own dataset with 1000 images.
What I did:
  - set flip_rate = 0 (all augmentation off)
  - set weight_decay = 0

I trained model for 20 epoch, but TPR for some classes very small, especially the number of which is small.
Do you use any other regularization?

Thank you for your work

**Comment by hellock** [comment]
No other regularization is used. You may try tuning the hyper-parameters like learning rate, etc.

**Comment by progred** [comment]
Thank you! I noticed that after adding BatchNorm into Resnet it got a little better.
